pxteditor/editor.ts

```diff
@@ -330,7 +330,7 @@ namespace pxt.editor {
         resourceImporters?: IResourceImporter[];
         beforeCompile?: () => void;
         patchCompileResultAsync?: (r: pxtc.CompileResult) => Promise<void>;
-        deployCoreAsync?: (r: pxtc.CompileResult) => Promise<void>;
+        deployAsync?: (r: pxtc.CompileResult) => Promise<void>;
```

Comment:
why are we renaming this field?

Reply:
Do we have a clean delineation of core <-> target contracts? How do we know what an editor might depend on?